spec.html

```diff
@@ -28130,273 +28130,144 @@ <h1>Function Properties of the Math Object</h1>
+            1. If _ny_ is *-&infin;*, return an implementation-approximated value representing -&pi; / 2.
+          1. If _ny_ is *+&infin;*, then
+            1. If _nx_ is *+&infin;*, return an implementation-approximated value representing +&pi; / 4.
+            1. If _nx_ is *-&infin;*, return an implementation-approximated value representing +3&pi; / 4.
+          1. If _ny_ is *-&infin;*, then
+            1. If _nx_ is *+&infin;*, return an implementation-approximated value representing -&pi; / 4.
+            1. If _nx_ is *-&infin;*, return an implementation-approximated value representing -3&pi; / 4.
+          1. Return an implementation-approximated value representing the result of the arc tangent of the quotient _ny_ / _nx_.
+        </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-math.cbrt">
         <h1>Math.cbrt ( _x_ )</h1>
         <p>Returns an implementation-approximated value representing the result of the cube root of _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-        </ul>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *+0*, _n_ is *-0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+          1. Return an implementation-approximated value representing the result of the cube root of _n_.
+        </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-math.ceil">
         <h1>Math.ceil ( _x_ )</h1>
         <p>Returns the smallest (closest to *-&infin;*) Number value that is not less than _x_ and is an integer. If _x_ is already an integer, the result is _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-          <li>
-            If _x_ is less than 0 but greater than -1, the result is *-0*.
-          </li>
-        </ul>
-        <p>The value of `Math.ceil(x)` is the same as the value of `-Math.floor(-x)`.</p>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *+0*, _n_ is *-0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+          1. If _n_ &lt; 0 and _n_ &gt; -1, return *-0*.
+          1. If _n_ is an integer, return _n_.
```

Comment:
We try to use the term "integral Number" for Number values that represent a mathematical integer.

```suggestion
          1. If _n_ is an integral Number, return _n_.
```